modules/@apostrophecms/area/ui/apos/components/AposAreaWidget.vue

```diff
@@ -266,6 +279,14 @@ export default {
     };
   },
   computed: {
+    // Passed only to the preview layer (custom preview components).
+    followingValuesWithParent() {
+      return Object.entries(this.followingValues || {})
+        .reduce((acc, [ key, value ]) => {
+          acc[`<${key}`] = value;
```

Comment:
... Because you want the syntax to be `<title` if the widget wants to follow the title of the document, because it is a child of the area not the area itself, and this leaves the namespace clean if we want to follow something about the area itself someday. OK.

Reply:
Precisely. Area follows `title`, the widget schema fields see `<title`, here we pass again `<title` to the widget Preview component for consistency (and to prevent namespace collisions).